guidelines/releases-guidelines/01-release-strategy.md

```diff
@@ -98,9 +126,17 @@ After reaching the end of the development cycle, the Kyma developers create a re
 - two working days for minor releases
 - one week for major releases
 
-If during testing a release candidate any **critical issues** are identified, the issues should be documented in GitHub and communicated to both Release Master and Release Manager as soon as possible. The Release Master and Release Manager should be informed about the impact of the issue, time to fix, and any potential implications when fixing it. Not later than when the fix is implemented and merged into the master branch, the Release Manager makes a decision whether or not to cherry-pick the fix to the release branch.
+All Release Candidates will always be fully tested by all automated and manual tests defined, following the [test strategy](/guidelines/technical-guidelines/07-test-strategy.md). Manual tests are tracked in [this sheet](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88/edit) for each release candidate.
+
+During the release process, only [critical issues](#critical-issues) will be considered for cherry-picking (as described above). Only when all tests have passed, and no critical issues are blocking the release, will the Release Manager approve the final release to be prepared and published.
+
+If any issue is still blocking the release by the time it should be made available according to the release schedule, the Release Manager will decide to:
+
+- remove or rollback the change causing the issue
+- wait for a fix (possibly delaying the release)
+- skip the release completely
 
-Once testing and validation is complete with no more issues identified, the final release is prepared and executed.
+Such decision will be communicated to the community in the public **release** Slack channel.
```

Comment:
```suggestion
The Release Manager communicates such a decision to the community on the public **release** Slack channel.
```

Reply:
The active voice would help to avoid confusion as to who is responsible for that.